### ISSUE TITLE
MacOS 15 can not run. 

### ISSUE THREAD

**Issue by cevin** [saturneric/GpgFrontend#181]
```text
-------------------------------------
Translated Report (Full Report Below)
-------------------------------------

Process:               GpgFrontend [78578]
Path:                  /Applications/GpgFrontend.app/Contents/MacOS/GpgFrontend
Identifier:            com.bktus.gpgfrontend
Version:               2.1.4 (2.1.4)
Code Type:             ARM-64 (Native)
Parent Process:        launchd [1]
User ID:               501

Date/Time:             2024-09-23 08:56:28.1914 +0800
OS Version:            macOS 15.0 (24A335)
Report Version:        12
Anonymous UUID:        ****-****-****-****-****

Sleep/Wake UUID:       ****-****-****-****-****

Time Awake Since Boot: 370000 seconds
Time Since Wake:       111 seconds

System Integrity Protection: enabled

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000003a10
Exception Codes:       0x0000000000000001, 0x0000000000003a10

Termination Reason:    Namespace SIGNAL, Code 11 Segmentation fault: 11
Terminating Process:   exc handler [78578]

VM Region Info: 0x3a10 is not in any region.  Bytes before following region: 4311254512
      REGION TYPE                    START - END         [ VSIZE] PRT/MAX SHRMOD  REGION DETAIL
      UNUSED SPACE AT START
--->  
      __TEXT                      100f8c000-101280000    [ 3024K] r-x/r-x SM=COW  /Applications/GpgFrontend.app/Contents/MacOS/GpgFrontend

Thread 0 Crashed::  Dispatch queue: com.apple.main-thread
0   QtCore                        	       0x103cc1b0c QJsonDocument::isArray() const + 8
```

**Comment by saturneric** [comment]
Please refer to: https://github.com/saturneric/GpgFrontend/issues/173

**Comment by saturneric** [comment]
Fixed in v2.1.5.
